### PR TITLE
Assorted updates

### PR DESCRIPTION
**Pull request by XhmikosR**
@joscha let me know if you want me to change something :)

PS. have you thought about moving the package under the [dependents](https://github.com/orgs/dependents/repositories) GitHub organization?
PPS. I also have https://github.com/XhmikosR/node-detective-postcss/compare/dev...dev-major if you want to make a PR from that later

### CODE REVIEW

**Review by joscha — APPROVED** [review]
Thanks for this! We can most definitely move the package! Are owning this org?

**Comment by XhmikosR** [comment]
BTW this fixes all the dependabot PRs :)

Yup, I just added you to the org so you can move the repo yourself.

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Supports `@import` and [`@value ... from`](https://github.com/css-modules/postcs
 
 `npm install --save detective-postcss`
 
-It's the CSS (PostCSS dialect) counterpart to [detective](https://github.com/substack/node-detective), [detective-amd](https://github.com/mrjoelkemp/node-detective-amd), [detective-es6](https://github.com/mrjoelkemp/node-detective-es6), [detective-sass](https://github.com/mrjoelkemp/node-detective-sass), [detective-scss](https://github.com/mrjoelkemp/node-detective-scss).
+It's the CSS (PostCSS dialect) counterpart to [detective](https://github.com/substack/node-detective), [detective-amd](https://github.com/dependents/node-detective-amd), [detective-es6](https://github.com/dependents/node-detective-es6), [detective-sass](https://github.com/dependents/node-detective-sass), [detective-scss](https://github.com/dependents/node-detective-scss).
```
Comment (joscha) [inline README.md]:
```suggestion
It's the CSS (PostCSS dialect) counterpart to [detective](https://github.com/browserify/detective), [detective-amd](https://github.com/dependents/node-detective-amd), [detective-es6](https://github.com/dependents/node-detective-es6), [detective-sass](https://github.com/dependents/node-detective-sass), [detective-scss](https://github.com/dependents/node-detective-scss).
```

**Comment by joscha** [comment]
> Yup, I just added you to the org so you can move the repo yourself.

seems like you've done a few of these moves - anything I need to watch out for?

**Comment by XhmikosR** [comment]
There shouldn't be any issue. Just transfer the repo, update any links and cut a new patch release.

**Comment by XhmikosR** [comment]
https://github.com/joscha/node-detective-postcss/compare/master...XhmikosR:dev-major minus the last patch which should land separately for a major bump :)

**Comment by XhmikosR** [comment]
@joscha friendly ping :)

**Comment by joscha** [comment]
Ah, sorry about that. Haven't gotten to release or moving the repo yet. But
I accepted the invite, so the start has been made - will try to do this in
the coming days! Cheers!

**Comment by XhmikosR** [comment]
Cool, ping me when you move the repo but before you make a release so that we land a few other patches.

**Comment by joscha** [comment]
this was released as 5.1.0

**Pull request merged**